CorsixTH/Lua/entities/patient.lua

```diff
@@ -23,6 +23,7 @@ class "Patient" (Humanoid)
 
 ---@type Patient
 local Patient = _G["Patient"]
+local go_home_reasons = {CURED = "cured", KICKED = "kicked", OVER_PRICED = "overpriced"}
```

Comment:
There isn't any reason for this table to exist in lua, unlike C it doesn't add any type safety since you can refer to unset keys, and also unlike C it's not a free call. Ultimately you're saying go_home_reasons["CURED"] = "cured". You could achieve a little bit of type safety with something like go_home_reasons.CURED = {}, but it's probably best to leave the reason as a string and document the possible values on goHome().